### PR TITLE
updating the UI for issue #120 and #118

### PR DESCRIPTION
**Pull request by jenbreese**
Please pull in the latest changes.

### CODE REVIEW

File: ui/app/search/results-dir.html [hunk 1]
```diff
@@ -1,15 +1,23 @@
...
         <span class="highlight" ng-if="text.highlight !== undefined">{{text.highlight}}</span>
-        <span class="features"><strong>Features:</strong> list, of, features</span>
-        <span class="browsers"><img src="/images/icons/chrome.png"/><img src="/images/icons/firefox.png"/><img src="/images/icons/ie.png"/><img src="/images/icons/safari.png"/></span>
       </span>
     </div>
+      <div class="features">
+        <span class="features" ng-show="result.metadata.feature.values.length > 0"><strong>Features:</strong>
+        <span ng-repeat="value in result.metadata.feature.values">{{value}}<span ng-hide="$last">, </span></span>
+ </span>
+        <!-- <em>Features</em> -->
```
Comment (grtjn) [inline ui/app/search/results-dir.html]: I think these commented lines can be deleted. You can append to your pull request by committing and pushing more changes.

File: ui/app/views/demo.html [hunk 2]
```diff
@@ -64,7 +64,7 @@ <h2 class="col-md-12">{{model.demo.name}} <a class="btn btn-primary btn-xs" ng-h
         <autocomplete ng-model="model.additionalBug.assignee" data="model.users" click-activation="true" ></autocomplete>
       </fieldset>
       <div class="col-md-12">
-        <a class="col-md-1" href ng-click="resetBugForm(); showBugForm = false">Cancel</a>
+        <a class="btn btn-default btn-xs" href ng-click="resetBugForm(); showBugForm = false">Cancel</a>
```
Comment (grtjn) [inline ui/app/views/demo.html]: Funny, the Cancel is an 'a', but Add a 'button'. Because of default behavior maybe?

**Comment by grtjn** [comment]
Also fixes #118..

**Comment by jenbreese** [comment]
Ok, removed the commented lines and pushed. I left cancel as an <a> and just styled to get the button appearance.

File: ui/app/views/demo.html (hunk 2 above)
Comment (jenbreese) [inline ui/app/views/demo.html]: The Cancel was an 'a' so I left it and just added the btn styles. The classes are designed for a, button, or input.

Comment (grtjn) [inline ui/app/views/demo.html]: No problem, just wondering..

**Pull request merged**